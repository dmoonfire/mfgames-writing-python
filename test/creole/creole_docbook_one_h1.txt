= Heading 1

This is the first paragraph in the file.
