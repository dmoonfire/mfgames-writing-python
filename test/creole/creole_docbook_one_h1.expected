<?xml version="1.0" encoding="UTF-8"?><article xmlns='http://docbook.org/ns/docbook' xmlns:mw='http://mfgames.com/mfgames-writing' version="5.0"><info>Heading 1</info><para>This is the first paragraph in the file.</para></article>